flanker-test-app/src/main/assets/task/Flanker.json

```diff
@@ -20,33 +20,605 @@
+      ]
+    },
+    {
+      "identifier" : "T.1I",
+      "text" : "Now let's try some more. Work as fast as you can without making mistakes.",
+      "type" : "instruction",
+      "actions"        : { "goForward": { "type":"default", "buttonTitle": "Next" }},
+      "shouldHideActions" : ["goBackward"],
+      "viewTheme"      : { "viewIdentifier":"FlankerAdditionalInstructions", "storyboardIdentifier":"Flanker" },
+      "nextStepIdentifier": "P.1"
+    },
+    {
+      "identifier" : "T.1L",
+      "type" : "instruction",
+      "text" : "Now we're going to take the Arrows test. Click \"Repeat Instructions\" if you want to repeat the instructions for this test. Otherwise click the \"Begin Test\" button to start the test.",
+      "actions"        : { "goForward": { "type":"default", "buttonTitle": "Begin Test" }, "goBackward": { "type":"default", "buttonTitle": "Repeat Instructions" }},
+      "viewTheme"      : { "viewIdentifier":"FlankerAdditionalInstructions", "storyboardIdentifier":"Flanker" },
+      "stepName":"repeatInstruction",
+      "stepBackTo": "firstInstruction",
+      "inputFields" : [
+        {
+          "identifier" : "T.1L",
+          "type" : "singleChoice",
+          "choices" : [
+            {
+              "itemResponseOID" : "E3BF01F4-1BBD-4A96-9E95-75C04A0F0D92",
+              "text" : "Repeat Instructions",
+              "value" : 1
+            },
+            {
+              "itemResponseOID" : "319DADAE-AAB0-4464-B2D8-A14E131DB116",
+              "text" : "Begin Test",
+              "value" : 2
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "identifier" : "T.2L",
+      "text" : "Remember to click the button pointing the same direction as the center arrow. Work as fast as you can without making mistakes.",
+      "type" : "instruction",
+      "actions"        : { "goForward": { "type":"default", "buttonTitle": "Next" }},
+      "shouldHideActions" : ["goBackward"],
+      "viewTheme"      : { "viewIdentifier":"FlankerAdditionalInstructions", "storyboardIdentifier":"Flanker" },
+      "branchingNavigationRules" : [
+        {
+          "description": "If last administration was more than a day ago than administer P.1",
+          "nextStepName": "firstPracticeInstruction",
+          "criteria":[
+            {
+              "name": "lastAdministrationGreaterThanDays",
+              "description": "If last administration was more than 1 day",
+              "duration":1
+            }
+          ]
+        },
+        {
+          "description": "If last administration was within a day and go to live step",
+          "nextStepName": "firstLiveStep",
+          "criteria":[
+            {
+              "name": "lastAdministrationLessThanEqualToDays",
+              "description": "If last administration was within a day",
+              "duration":1
+            }
+          ]
+        }
+      ]
+
+    },
+    {
+      "identifier" : "P.1",
+      "type" : "flankerForm",
```

Comment:
I did end up changing the JSON a bit. Substituted `flankerForm` for `form`. I can probably get it to work with `form`, but this was a bit faster. Let me know when/if this needs to be done

Reply:
What would it take to get it to work with `form`?